App/src/services/auth.ts

```diff
@@ -25,7 +41,7 @@ export function login(dto: LoginDTO): Promise<AuthenticationResponse> {
     return axios.post("/Authentication/Login", dto);
 }
 
-export function register(dto: RegisterDTO): Promise<AuthenticationResponse> {
+export function registeration(dto: RegisterDTO): Promise<AuthenticationResponse> {
```

Comment:
why did you change the name here?

Reply:
Another constant shared the name, I changed it so there wasn't a conflict because of a function and name having the same name, and the constant defined was used more than this function so this was changed

Reply:
I would say name it something like RegisterUser() or RegisterNewUser(). just because methods/functions do something so they are usually verbs.

Reply:
I agree with Marc that `RegisterNewUser` (also registeration is a typo).